src/supply.ts

```diff
@@ -92,17 +92,18 @@ const getCompanyCirculating = async () => {
 
 const getPreSale = async () => {
     const TOTAL_PRESALE = 387917402
+    const TOTAL_UNLOCKS = 8
 
     const truContract = contractAt('TrustToken', contracts.tru)
-
     const toBeDistributed = await truContract.balanceOf(contracts.preSaleToBeDist)/1e8
     const registeredButUnclaimed = await truContract.balanceOf(contracts.preSaleRegisteredUnclaimed)/1e8
     const unclaimedMultiSig = await truContract.balanceOf(contracts.preSaleUnclaimedMultiSig)/1e8
     const alreadyDist = TOTAL_PRESALE-toBeDistributed-registeredButUnclaimed-unclaimedMultiSig
 
     const FIRST_RELEASE_DATE = Date.parse('21 Nov 2020 00:00:00 GMT')
-    const numberOfRelease = Math.ceil((Date.now()-FIRST_RELEASE_DATE)/(1000*60*60*24*30*3))
-    const preSale = alreadyDist * numberOfRelease/8
+    const releasePeriods = Math.ceil((Date.now()-FIRST_RELEASE_DATE)/(1000*60*60*24*30*3))
+    const unlocks = Math.min(releasePeriods, TOTAL_UNLOCKS)
+    const preSale = alreadyDist * unlocks/TOTAL_UNLOCKS
```

Comment:
Because we are past the unlock period we can probably do away with this and simplify the logic a bit -- will prep this as a separate PR